roles/pulp-database/tasks/main.yml

```diff
@@ -35,7 +35,7 @@
         name: '{{ result.files[0].path }}/site-packages/pulpcore/app/migrations'
         state: directory
         owner: '{{ pulp_user }}'
-        group: '{{ pulp_user }}'
+        group: pulp
```

Comment:
It's a good practice to not hardcode things. I'd introduce a `pulp_group` variable and use that.